README

```diff
@@ -11,10 +11,12 @@ Requirements:
 	Unless you have some other reasons for wanting to use the svn version,
 	it is best to install the latest release (3.9).
 	For more details, see pet/README.
+- zlib1g-dev Compression library
```

Comment:
This is the name of the Ubuntu package, not the name of the project clang depends on.

Reply:
Ok, i'll fix the package name, i hadn't paid attention to this detail.